pelita/ui/tk_canvas.py

```diff
@@ -895,8 +892,6 @@ def draw_status_info(self, game_state):
                 else:
                     self.ui_bot_traffic_lights_canvas.itemconfig(circle, fill="#bbb")
 
-        layout_name = "–" if game_state.get("layout_name") is None else game_state.get("layout_name")
```

Comment:
I think we might add the seed now to the UI? (Possibly together with the food specification.) We should make it possible to recreate a maze from a screenshot, if only for dealing with bug reports. I don’t think it needs to be as prominent as the layout name, though.

Reply:
We tried it. We convinced ourselves that it was not useful. To be useful in anyway the text from the GUI would have to be selectable so that you can copy it with ctrl+c, which is not. Typing the seed from a screenshot is not going to be nice either. I don't think any of this is a problem. The seed is printed very prominently in the CLI, and you never use pelita without the CLI.
I am not sure it is worth it to try to make the GUI a sort of `repr` of the game. The CLI is good enough for this purpose in my opinion.

Reply:
Right, makes sense. Although on macOS any text is now selectable in a screenshot, so it is not too much of a problem.

If we find that we need it, we may still add it later.